docs/modules/ROOT/pages/troubleshooting/debugging.adoc

```diff
@@ -52,3 +53,12 @@ image::debugging/remote-debugger.png[Configuration of the remote debugger in Int
 Once you configure a debugger, you can **add breakpoints** to various part of the code, then connect the debugger to trigger the JVM startup.
 
 When the debugging session is finished, hitting *ctrl+c* on the terminal where the kamel CLI is running will restore the integration to its original status.
+
+[[debugging-kamelets]]
+== Debugging Kamelets
+
+As we've seen in the previous section, all `Integration` created in Camel K are finally bundled as a Java application, hence, the possibility to debug via JVM debugger. Any `Kamelet` you will be using directly in your `Route` definition or in a `KameletBinding` is automatically converted in a `yaml` route and injected in the Camel Context to be executed. That means that you cannot directly debug a `Kamelet` as you would do with a Java or any other JVM language `Route`.
+
+However, you can troubleshoot individually each `Kamelet` definition by focusing on the specification xref:kamelets/kamelets-user.adoc#_flow[`Flow`]. As an example, you can create a simple https://camel.apache.org/camel-k/latest/languages/yaml.html[`yaml`] test `Route` substituting the `kamelet:source` or `kamelet:sink` with any mock endpoint that can help you debugging the single `Kamelet` flow. Even using a `timer` and a `log` component may be enough for a basic check.
```

Comment:
Should a `link:` or `xref:` be used https://camel.apache.org/camel-k/latest/languages/yaml.html?

Reply:
I thought the same. However these languages seem to be added later in the process by the merge with main Camel, so, I am not sure if I can still reference via `link:` or `xref:`

Reply:
@zregvart @oscerd would you know if there is a way to use `link:` of `xref:` pointing to pages from external modules (https://github.com/apache/camel-k/blob/main/docs/antora.yml#L6), or raw links can be safely used in that case?

Reply:
`xref` needs to be used here here, absolute links are not allowed to camel.apache.org domain. See the [xref reference](https://docs.antora.org/antora/2.3/page/version-and-component-xrefs/).

```suggestion
However, you can troubleshoot individually each `Kamelet` definition by focusing on the specification xref:kamelets/kamelets-user.adoc#_flow[`Flow`]. As an example, you can create a simple xref:camel-k:languages:yaml.adoc[`yaml`] test `Route` substituting the `kamelet:source` or `kamelet:sink` with any mock endpoint that can help you debugging the single `Kamelet` flow. Even using a `timer` and a `log` component may be enough for a basic check.
```

Reply:
Thanks @zregvart!

Reply:
Thanks @zregvart, I've committed your suggestion. @astefanutti I think we can merge it now, thanks!